chain/cmd/bandsv/main.go

```diff
@@ -18,7 +30,130 @@ type OracleRequestResp struct {
 	Params    []byte `json:"params"`
 }
 
+type TxResult struct {
+	Hash   string `json:"hash"`
+	Height string `json:"height"`
+	Index  int    `json:"index"`
+	Result struct {
+		Code      int         `json:"code"`
+		Data      interface{} `json:"data"`
+		Log       string      `json:"log"`
+		Info      string      `json:"info"`
+		GasWanted string      `json:"gasWanted"`
+		GasUsed   string      `json:"gasUsed"`
+		Events    []struct {
+			Type       string `json:"type"`
+			Attributes []struct {
+				Key   string `json:"key"`
+				Value string `json:"value"`
+			} `json:"attributes"`
+		} `json:"events"`
+		Codespace string `json:"codespace"`
+	} `json:"tx_result"`
+	Tx string `json:"tx"`
+}
+
+type CodeResponse struct {
+	Response struct {
+		Code      int         `json:"code"`
+		Log       string      `json:"log"`
+		Info      string      `json:"info"`
+		Index     string      `json:"index"`
+		Key       string      `json:"key"`
+		Value     interface{} `json:"value"`
+		Proof     interface{} `json:"proof"`
+		Height    string      `json:"height"`
+		Codespace string      `json:"codespace"`
+	} `json:"response"`
+}
+
+const priv = "a96e62ed3955e65be3aaa3f12d87b6b5cf26039ecfa948dc5107a495418e5430"
+const port = "5001"
+const nodeURI = "http://localhost:26657"
+const queryURI = "http://localhost:1317"
+
+func GetTx(txHash string) (TxResult, error) {
+	_resp, err := http.Get(fmt.Sprintf("%s/tx?hash=0x%s", nodeURI, txHash))
```

Comment:
Instead of using `http.Get` and need to raw stuff like `ioutil.ReadAll`, let's use convenient library like grequests? https://github.com/levigross/grequests

Reply:
In fact, instead of using `http.Get` or grequests, can't we use Tendermint's rpc client directly? See https://github.com/tendermint/tendermint/blob/master/lite2/rpc/client.go for example. It really seems like all the code is written for us already.